app/Http/Controllers/Store/CentreController.php

```diff
@@ -234,6 +235,23 @@ private function getCentreFamilyRegistrationsSummary(array $centre_ids, $dateFor
             // Add our kids back in
             $row = array_merge($row, $kids);
 
+            // Calculate number of days on programme
+            if ($reg->family->leaving_on && !$reg->family->rejoin_on) {
+                $startDate = Carbon::parse($reg->created_at);
+                $leaveDate = Carbon::parse($reg->family->leaving_on);
+                $diff = $startDate->diffInDays($leaveDate);
+            } elseif ($reg->family->rejoin_on) {
```

Comment:
Is there any chance they might not have a leaving_on value? If there is, then you probably need to check it exists as well, so you don't try and do a parse on null.

Reply:
If they have a rejoin_on, they will have a leaving_on value because they have left once and then rejoined